rotkehlchen/exchanges/kraken.py

```diff
@@ -758,9 +785,6 @@ def query_online_deposits_withdrawals(
                 fee = Fee(ZERO)
 
             amount = movement.balance.amount
-            if movement.event_type == HistoryEventType.WITHDRAWAL:
```

Comment:
What is the reason for removing this?

Reply:
The amount is already positive after the changes So when building the withdrawal event we don't need to make the absolute value

Reply:
The amount is already positive after the changes So when building the withdrawal event we don't need to make the absolute value